pallets/rmrk-core/src/lib.rs

```diff
@@ -81,6 +88,26 @@ pub mod types;
 // Re-export pallet items so that they can be accessed from the crate namespace.
 pub use pallet::*;
 
+#[cfg(feature = "runtime-benchmarks")]
+pub trait BenchmarkHelper<CollectionId, ItemId> {
+	fn collection(i: u16) -> CollectionId;
```

Comment:
I haven't implemented much benchmarking in the past, but why do we opt to use `u16` in benchmarking? Is this a best practice?

Reply:
It is a copy from the Uniques pallet. But it can be `u32` as well. In fact it would be more clear if it is `u32` like it is defined in the runtime.